demo.py

```diff
@@ -0,0 +1,156 @@
+# %% [markdown]
+# # Welcome to the demo of Skore!
+#
+# Let's start by fetching the dataset. We will use the census dataset from openml.
+#
+# It's a binary classification problem, where the target is whether a person earns more than 50K a year.
+#
+# https://www.openml.org/search?type=data&sort=runs&id=1590&status=active
+
+# %%
+from sklearn.datasets import fetch_openml
+
+X, y = fetch_openml("adult", version=2, as_frame=True, return_X_y=True)
+# %%
+# Let's take a look at the data
+# in real life, we would do a lot more data exploration.
+X.info()
+# %%
+y.value_counts()
+
+# %%
+from sklearn.preprocessing import LabelEncoder
+
+le = LabelEncoder()
+y_encoded = le.fit_transform(y)
+
+# %%
+import pandas as pd
+
+pd.Series(y_encoded).value_counts()
+
+# %%
+from sklearn.model_selection import train_test_split
+
+X_train, X_test, y_train, y_test = train_test_split(X, y_encoded, random_state=1)
+
+# %% [markdown]
+# Simpler is better.
+#
+# Let's do a simple baseline.
+
+# %%
+from skrub import tabular_learner
+
+baseline = tabular_learner("classification")
+baseline
+# %%
+from skore import EstimatorReport
+
+baseline_report = EstimatorReport(
+    baseline,
+    X_train=X_train,
+    y_train=y_train,
+    X_test=X_test,
+    y_test=y_test,
+)
+baseline_report.help()
+
+# %%
+baseline_report.metrics.report_metrics()
+
+# %% [markdown]
+# Let's go a bit further in that baseline by optimizing the parameters.
+
+# %%
+from sklearn.model_selection import GridSearchCV
+
+tuned_baseline = GridSearchCV(
+    estimator=baseline,
+    param_grid={
+        "histgradientboostingclassifier__learning_rate": [0.01, 0.1, 0.2],
+        "histgradientboostingclassifier__max_depth": [1, 3, 5],
+        "histgradientboostingclassifier__max_leaf_nodes": [30, 60, 90],
+    },
+    cv=5,
+    n_jobs=-1,
+    refit=True,
+    scoring="neg_log_loss",
+)
+tuned_baseline
+
+# %%
+tuned_baseline_report = EstimatorReport(
+    tuned_baseline, X_train=X_train, y_train=y_train, X_test=X_test, y_test=y_test
+)
+# %%
+tuned_baseline_report.metrics.report_metrics()
+# %%
+from skore import ComparisonReport
+
+comp = ComparisonReport(
+    {"Baseline Model": baseline_report, "Tuned model": tuned_baseline_report}
+)
+comp.help()
+# %%
+comp.metrics.report_metrics(pos_label=1, indicator_favorability=True)
+
+# %%
+# create or connect to project
+from skore import Project
+
+project = Project("hub://Probabl/project demo - census")
+# %%
+project.put("baseline", baseline_report)
+project.put("tuned_baseline", tuned_baseline_report)
+
+# %% [markdown]
+# # DEMO PART 2 - after supervisor review
+#
+# Their request: even more simple baselines: dummy classifier, and a linear model.
+
+# %%
+from sklearn.dummy import DummyClassifier
+
+dummy = DummyClassifier(strategy="most_frequent")
```

Comment:
Actually, `"prior"` is also doing the same than `most_frequent` for the `predict`. So if you wish to simplify, you can keep the default. It should not change anything in the demo.